libs/schemas/src/queries/thread.schemas.ts

```diff
@@ -166,6 +166,14 @@ export const DEPRECATED_GetBulkThreads = z.object({
   withXRecentComments: z.coerce.number().optional(),
 });
 
+export const GetThreadsByIds = {
+  input: z.object({
+    community_id: z.string(),
+    thread_ids: z.string(),
+  }),
+  output: z.array(Thread),
```

Comment:
Technically since some joins are marked as `required: true`, sub-properties that are marked as optional in the Thread schema will always be included in the output e.g. `Thread.Address`. We should figure out a clean way to ensure that the output matches the required joins so we don't need to handle the undefined case on the client.

Reply:
See commit https://github.com/hicommonwealth/commonwealth/pull/9660/commits/98dde3387bcaec1e92fec1759a3ed6e2a409975e for more ThreadView reconciliation pattern applied to all thread queries

Reply:
I like this pattern but it doesn't address the specific issue, right? `Address` in the returned object type is still optional even though it will always be included.

Reply:
Yeah, we still have a few loose ends in the "View" types,  that should be incrementally fixed. This PR is just to continue establishing the pattern at the Thread level, which is the big one... I don't want to keep extending the PR to fix everything